src/sortable.h

```diff
@@ -33,12 +31,10 @@ extern "C" {
 /* RSSortingVector is a vector of sortable values. All documents in a schema where sortable fields
  * are defined will have such a vector. */
 typedef struct RSSortingVector {
-  unsigned int len : 8;
+  unsigned char len;
   RSValue *values[];
 } RSSortingVector;
 
-#pragma pack()
```

Comment:
Won't it increase `sizeof(RSSortingVector)` with padding (depending on the alignment - could even be a 7 byte increase? per document?)

Reply:
This actually shrinks it size. It was 4 bytes with 3 of them for padding since we limit the `Len` parameter to 8 bits but didn’t use them for anything else. The flexible array has a size of 0 regardless, and the packing pragma didn’t affect the struct.
Now we have a struct of size 1 without the need of a pragma

Reply:
I returned the packing. making sure we got the byte saving